src/SSVOpenHexagon/Data/StyleData.cpp

```diff
@@ -123,18 +134,20 @@ void StyleData::computeColors()
 
     if(currentColors.size() > 1)
     {
-        ssvu::rotate(currentColors,
-            begin(currentColors) + currentSwapTime / (maxSwapTime / 2.f));
+        ssvu::rotate(currentColors, begin(currentColors) +
+                                        currentSwapTime / (maxSwapTime / 2.f) +
+                                        colorPosOffset % levelStatus.sides);
     }
 }
 
-void StyleData::drawBackground(sf::RenderTarget& mRenderTarget,
-    const sf::Vector2f& mCenterPos, const LevelStatus& levelStatus) const
+void StyleData::drawBackground(RenderTarget& mRenderTarget,
+    const sf::Vector2f& mCenterPos, LevelStatus& levelStatus,
```

Comment:
Not sure why you got rid of the `const` on `levelStatus` here. Immutability is good to avoid mistakes and help readability

Reply:
huh, well, I actually didn't mean to not set it as constant